runtime/src/bank.rs

```diff
@@ -2190,11 +2181,7 @@ impl Bank {
             drop_callback: RwLock::new(OptionalDropCallback(None)),
             freeze_started: AtomicBool::new(fields.hash != Hash::default()),
             vote_only_bank: false,
-            cost_tracker: RwLock::new(CostTracker::new_with_account_data_size_limit(
-                feature_set
-                    .is_active(&feature_set::cap_accounts_data_len::id())
-                    .then(|| MAX_ACCOUNTS_DATA_LEN.saturating_sub(accounts_data_size_initial)),
-            )),
```

Comment:
Up at line 2138 you can see `feature_set` is `default()`. So this check is always false here. Now that the check is in `finish_init()`, which this fn calls later, the code highlighted here can be removed.